localization/english/wc_regiment_l_english.yml

```
@@ -14,7 +14,155 @@
+ dervishes_flavor:0 "#F Raiders, bandits, and fearsome warriors, the Wastewander Dervishes are a force that have taken the desert and made it their own."
+ dire_troll_flavor:0 "#F Strikingly large and powerful trolls, there are few who can match a dire troll in raw strength, leading to their reputation as elite warriors."
+ dragon_flavor:0 "#F Forged by the titans, wielders of incredible powers, and rulers of the sky, only a fool would challenge a dragon."
+ ghoul_flavor:0 "#F Ghouls are undead acting as the shock troops for the Scourge. These lumbering, rotting corpses were once innocent townsfolk who have made the final transition into true undeath."
+ gladiator_flavor:0 "#F Gladiators are raised and trained for life and battle in the arena. This allows them to bring their superior tactics to bear on the battlefield as well."
+ gryphon_rider_flavor:0 "#F The daring Gryphon Riders charge into battle on their noble beasts with relentless determination to keep the heavens clear of their enemies."
+ hexer_flavor:0 "#F The capability of Hexers with chemicals and magic is surpassed only by their brutality."
+ headhunter_flavor:0 "#F Dangerous troll hunters, their expertise makes them formidable trackers, spearthrowers, and scouts."
+ marauder_flavor:0 "#F The marauders are centaur warriors who exemplify speed, strength, and savagery, making them a fearsome opponent."
+ magi_flavor:0 "#F The two-headed ogre magi throw themselves into battle with a wild array of spells. If magic is not enough, they often make use of their immense physical strength and crush unwary enemies with their bare fists."
+ mountain_giant_flavor:0 "#F The mighty mountain giants may be slow, but they can strike all the harder for it. These stone behemoths are extremely difficult to dissuade from a path once taken and often leave a trail of destruction in their wake."
+ myrmidon_flavor:0 "#F Larger and stronger than female Naga, Myrmidons appear quite beastly with their dragon-like heads and scale-covered bodies. The ability to operate in water and on land and to take on a variety of roles makes the Myrmidon the perfect cornerstone of any Naga unit."
+ savage_flavor:0 "#F The world believes us barbarians and monsters. Unfortunately, our most savage warriors do little to disprove this. Despite their flaws, they are highly capable warriors, nigh-unstoppable in battle."
+ sea_giant_flavor:0 "#F What they lack in loyalty, our algae-covered Sea Giant slaves more than make up for in strength. Few are the foes who would dare challenge them, and fewer still would succeed."
+ sentinel_flavor:0 "#F Being the main force of the Kaldorei, Sentinels adopt flexible strategies in combat, particularly favoring ambushes or hit-and-run tactics."
```

```suggestion
 sentinel_flavor:0 "#F Being the main force of the Kaldorei, Sentinels adopt flexible strategies in combat, particularly favoring ambushes and hit-and-run tactics."
```
I would change "or" to "and".